cmd/juju/application/refresher/refresher_test.go

```diff
@@ -745,6 +763,18 @@ func (s *charmHubCharmRefresherSuite) TestCharmHubResolveOriginEmptyTrackEmptyCh
 	}))
 }
 
+// type switchOriginSuite struct{}
```

Comment:
Change looks good. Just remember to either remove (or finish) this test before merging.